src/pages/rt-fees-schedule/forms/FeesSceduleForm.js

```diff
@@ -0,0 +1,145 @@
+// ** MUI Imports
+import { Grid } from '@mui/material'
+import { DataSets } from 'src/resources/DataSets'
+
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+
+// ** Custom Imports
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import FormShell from 'src/components/Shared/FormShell'
+
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useContext, useEffect } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { useForm } from 'src/hooks/form'
+import { RemittanceOutwardsRepository } from 'src/repositories/RemittanceOutwardsRepository'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+
+const FeesSceduleForm = ({ labels, maxAccess, setStore, store, onChange }) => {
+  const { postRequest, getRequest } = useContext(RequestsContext)
+  const { recordId } = store
+
+  const invalidate = useInvalidate({
+    endpointId: RemittanceOutwardsRepository.FreeSchedule.qry
+  })
+
+  const { formik } = useForm({
+    maxAccess,
+    initialValues: {
+      recordId: null,
+      name: '',
+      reference: '',
+      originCurrency: ''
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      name: yup.string().required(),
+      reference: yup.string().required(),
+      originCurrency: yup.string().required()
+    }),
+    onSubmit: values => {
+      postGroups(values)
+    }
+  })
+
+  useEffect(() => {
+    onChange(formik.values)
```

Comment:
we care about recordId only, why u're updating setStore whenever formik values changes?

Reply:
what if we want to add a store